src/main/kotlin/org/malachite/estella/organization/api/OrganizationController.kt

```diff
@@ -34,6 +34,21 @@ class OrganizationController(
     fun getOrganization(@PathVariable organizationId: OrganizationID): ResponseEntity<Organization> =
         ResponseEntity(organizationService.getOrganization(organizationId.toId()), HttpStatus.OK)
 
+    @CrossOrigin
+    @GetMapping("/organization")
+    fun getOrganizationByUser(@RequestHeader(EStellaHeaders.jwtToken) jwt: String?): ResponseEntity<Any> =
+        securityService.getOrganizationFromJWT(jwt)
+            ?.let { ResponseEntity.ok(it) }
+            ?: UNAUTH
+
+    @CrossOrigin
+    @GetMapping("/offers")
+    fun getOrganizationsOffers(@RequestHeader(EStellaHeaders.jwtToken) jwt: String?): ResponseEntity<Any> =
+        securityService.getOrganizationFromJWT(jwt)
+            ?.let { offerService.getOrganizationOffers(it) }
```

Comment:
Wyniósłbym to do odpowiednich serwisów. To do offer service i tam sprawdzał z security i getował oferty. Tamto wyższe też, tylko chyba user service będzie odpowiednim miejscem (albo i nie, to się okaże) - jak Ci się nie uda tego wyższego wynieść, to zostaw tak, poprawię w tym swoim (tam był problem z rekurencją jakąś user service - > security service). Ogólnie pamiętajmy, że kontrolery nie powinny mieć żadnej logiki albo minimalnie logiki (czyli Maks wywołanie jednej metody z jednego serwisu i mapowanie do dto)

Reply:
to w takim razie poczekam z tym na ES-147